tests/specs/serve/no_args/__test__.jsonc

```diff
@@ -0,0 +1,5 @@
+{
+  "args": "serve --port 12345 main.ts",
```

Comment:
Does `deno serve` find an open port? Maybe we should exclude it from the test? This port conflicts with another test.

Reply:
Is there a way to pick to an open port here? `deno serve` doesn't find an open port.

Reply:
This should fix it https://github.com/denoland/deno/pull/23846